fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -158,8 +158,11 @@ ThreadFuture<MappedRangeResult> DLTransaction::getMappedRange(const KeySelectorR
                                                               const KeySelectorRef& end,
                                                               const StringRef& mapper,
                                                               GetRangeLimits limits,
+                                                              int matchIndex,
                                                               bool snapshot,
                                                               bool reverse) {
+	// question: I added a parameter in fdb_transaction_get_mapped_range,
```

Comment:
@jzhou77 please advise here, `transactionGetMappedRange` cannot take one more parameter, while I changed`fdb_transaction_get_mapped_range` to take one more. 

from https://github.com/apple/foundationdb/blob/153187542522730f270f342c858ae2326ae770b0/fdbclient/MultiVersionTransaction.actor.cpp#L653-L654 we can see it is dynamically loading.

Reply:
let me try changing https://github.com/apple/foundationdb/blob/153187542522730f270f342c858ae2326ae770b0/fdbclient/MultiVersionTransaction.h#L206

Reply:
seems work!